test/unit/server/services/procedures/chart/chart.spec.js

```diff
@@ -32,11 +32,65 @@ describe(utils.suiteName(__filename), function() {
             'grid',
             'isTimeSeries',
             'timeInputFormat',
-            'timeDisplayFormat'
+            'timeDisplayFormat',
+            'logscale',
         ];
         assert.deepEqual(opts.map(i => i[0]), expectedOpts);
     });
 
+    describe('logscale', function() {
+        it('should allow empty axes', function(){
+            const [, options] = Chart._parseDrawInputs([], {logscale: ''});
+            assert.equal(options.logscale, undefined);
+        });
+        it('should allow empty axes', function(){
```

Comment:
Duplicate test name

Reply:
Also, do we want to allow an empty list as the logscale? I am not sure this is the desired behavior.

Reply:
I allowed it so that a "combine <list> using join(_,_)" block with a (potentially-empty) list of individual axis names would still be allowed (as a no-op). But I guess given how options work they could just conditionally add the logscale option itself.